app/models/manageiq/providers/openshift/container_manager/container_image.rb

```diff
@@ -0,0 +1,13 @@
+class ManageIQ::Providers::Openshift::ContainerManager::ContainerImage < ContainerImage
+  acts_as_miq_taggable
+  include NewWithTypeStiMixin
```

Comment:
These two lines should not be here.  They both belong in the base ContainerImage.